templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -305,7 +308,7 @@ contents:
 
       if ! nmcli connection show "$ovs_interface" &> /dev/null; then
         ovs-vsctl --timeout=30 --if-exists destroy interface "$bridge_name"
-        if [ "${ipv4_method}" = "manual" ] || [ "${ipv6_method}" = "manual" ]; then
+        if [ "${ipv4_method}" = "manual" ] || [ "${ipv4_addresses}" != "" ] || [ "${ipv6_method}" = "manual" ] || [ "${ipv6_addresses}" != "" ]; then
```

Comment:
Could you add a comment here explaining the use case being addressed (manual or dhcp + static)?